apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -3,18 +3,12 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Radium from 'radium';
 import color from '@cdo/apps/util/color';
-import {AnimationCategories} from '../gamelab/constants';
-import {CostumeCategories} from '../spritelab/constants';
-var msg = require('@cdo/locale');
-import animationLibrary from '../gamelab/animationLibrary.json';
-import spriteCostumeLibrary from '../spritelab/spriteCostumeLibrary.json';
+import msg from '@cdo/locale';
```

Comment:
(you'll know the answer to this better than i do, but) do we have a preferred naming convention for this import? sometimes it's `i18n` and sometimes its `msg`

Reply:
I don't think we do unfortunately. I chose `msg` here because we used it in all but one case in this file but overall I don't have a personal preference. I would love to standardize it a bit better though...